Libraries/Utilities/HMRClient.js

```diff
@@ -40,18 +40,36 @@ const HMRClient = {
 
     const activeWS = new WebSocket(wsUrl);
     activeWS.onerror = (e) => {
-      throw new Error(
+      let error = (
 `Hot loading isn't working because it cannot connect to the development server.
 
-Ensure the following:
-- Node server is running and available on the same network
-- run 'npm start' from react-native root
-- Node server URL is correctly set in AppDelegate
+Try the following to fix the issue:
+- Ensure that the packager server is running and available on the same network`
+      );
+
+      if (Platform.OS === 'ios') {
+        error += (
+`
+- Ensure that the Packager server URL is correctly set in AppDelegate`
+        );
```

Comment:
Can we make eslint either ignore these by `/* eslint-ignore quotes:0 */` or updating?